app/models/aliquot.rb

```diff
@@ -44,8 +44,8 @@ class InsertSize < Range
   # An aliquot is held within a receptacle
   belongs_to :receptacle, inverse_of: :aliquots
 
-  belongs_to :tag
-  belongs_to :tag2, class_name: 'Tag'
+  belongs_to :tag, optional: true
+  belongs_to :tag2, class_name: 'Tag', optional: true
 
```

Comment:
This doesn't actually change the behaviour. By default more recent versions of rails make belongs_to required by default. We're enabling the legacy setting elsewhere, but this will make it easier to adopt the new defaults in future.